### PR TITLE
PROFILES 4.1.2 - Changed tests and pluggable portion after DDF started sending POST SSO responses

### PR DESCRIPTION
**Pull request by blen-desta**
#### What does this PR do (if it's not clear from the Title)? Please include any specification references that apply.
Changed tests and the pluggable portion after DDF started correctly responding with POST SSO responses.

In the Profiles document under section 4.1.2 Profile Overview under step 5. `Identity Provider issues <Response> to Service Provider`:
```
The HTTP Redirect binding MUST NOT be used, as the response will typically exceed the 
URL length permitted by most user agents.
```
This PR is related to https://github.com/codice/ddf/pull/3188

#### Checklist:
- [ ] SAML Spec Table of Contents documentation updated

### CODE REVIEW

File: ctk/common/src/main/kotlin/org/codice/compliance/verification/core/SignatureSyntaxAndProcessingVerifier.kt [hunk 1]
```diff
@@ -31,7 +30,7 @@ class SignatureSyntaxAndProcessingVerifier(private val node: Node) {
 
     /** 5.4.2 References */
     private fun verifySignatureSyntaxAndProcessing() {
-        node.children(SSOConstants.SIGNATURE).forEach {
+        node.recursiveChildren(SSOConstants.SIGNATURE).forEach {
```
Comment (brjeter) [inline ctk/common/src/main/kotlin/org/codice/compliance/verification/core/SignatureSyntaxAndProcessingVerifier.kt]: Out of curiosity, how did you notice this issue?

Comment (blen-desta) [reply ctk/common/src/main/kotlin/org/codice/compliance/verification/core/SignatureSyntaxAndProcessingVerifier.kt]: @SmithJosh did

**Comment by blen-desta** [comment]
@bakejeyner, @brjeter is making those changes in his PR

**Comment by blen-desta** [comment]
build now

**Comment by blen-desta** [comment]
build now

**Comment by blen-desta** [comment]
build now
Should fail 🤞

**Comment by blen-desta** [comment]
🤔 3 tests failed

**Comment by blen-desta** [comment]
build now

**Comment by blen-desta** [comment]
build now

**Comment by brjeter** [comment]
Hero'd successfully

**Comment by blen-desta** [comment]
Merging without a successful build because DDF hasn't deployed yet. Tested locally.

**Pull request merged**